lib/active_model/serializer.rb

```diff
@@ -50,6 +50,10 @@ def self.type(type)
       self._type = type
     end
 
+    def self.links(&block)
+      self._links_block = block
+    end
```

Comment:
An other option would be to declare each link as a lambda, but that would make much more stored procs...